### ISSUE TITLE
Form.Input Joi.number is not a function

### ISSUE THREAD

**Issue by JennerChen** [freshesx/mogul#7]

**Comment by JennerChen** [comment]
rollup打包 joi-browser 会使用 _interopDefault 包装 require("joi-browser")导致问题
```
    function _interopDefault (ex) { return (ex && (typeof ex === 'object') && 'default' in ex) ? ex['default'] : ex; }
```

**Comment by JennerChen** [comment]
fixed in version [0.0.7](https://github.com/freshesx/mogul/tree/0.0.7)
https://github.com/freshesx/mogul/blob/d25698469e5b0abd8c073f6393eefff62d3970e1/src/components/Form/FormStore.js#L2-L3
